common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -18,6 +18,7 @@
 import org.apache.sedona.common.utils.GeometryGeoHashEncoder;
 import org.geotools.geometry.jts.JTS;
 import org.geotools.referencing.CRS;
+import org.jetbrains.annotations.Nullable;
```

Comment:
@SW186000 The test failed. Please fix the Nullable annotation from "org.jetbrains.annotations"

Reply:
Update the PR.